packages/client-core/src/components/InstanceChat/index.tsx

```diff
@@ -55,6 +55,10 @@ const InstanceChat = (props: Props): any => {
   const [unreadMessages, setUnreadMessages] = React.useState(false)
   const activeChannelMatch = Object.entries(channels).find(([key, channel]) => channel.channelType === 'instance')
   const instanceConnectionState = useLocationInstanceConnectionState()
+
+  const currentInstanceId = instanceConnectionState.currentInstanceId.value
+  const currentInstanceConnection = instanceConnectionState.instances[currentInstanceId!]
```

Comment:
is it possible for `instances` to not contain the given `currentInstanceId` ?

Reply:
yes

Reply:
For nullable states, it's generally a good idea to do this:

const currentInstanceConnection = instanceConnectionState.instances[currentInstanceId!].ornull